src/GitTabExpansion.ps1

```diff
@@ -472,6 +488,34 @@ function GitTabExpansionInternal($lastBlock, $GitStatus = $null) {
     }
 }
 
+function Expand-GitProxyCommand($Command) {
+    # Make sure the incoming command matches: <Command> <Args>, so we can extract the alias/command
+    # name and the arguments being passed in.
+    if ($Command -notmatch '^(?<command>\S+)([^\S\r\n]|[^\S\r\n]`\r?\n)+(?<args>([^\S\r\n]|[^\S\r\n]`\r?\n|\S)*)$') {
+        return $Command
+    }
+
+    # Store arguments for replacement later
+    $Arguments = $Matches['args']
```

Comment:
Local vars should be lower-case e.g `arguments`.  Also use `$matches['args']`.  We are mostly consistent on using $matches vs $Matches but I see there are three spots still using $Matches. Let's not add more.  :-)

Reply:
Makes sense.